traffic-participants.rst

```diff
@@ -0,0 +1,96 @@
+Modeling traffic participants
+==============================
+
+Traffic participants
+--------------------
+
+A traffic participant is an element of the simulated world, which can change its position and orientation during simulation time.
+A traffic participant represents one of the following:
+
+- A living being.
+- A means of transportation for living beings
+- A means of transportation for goods
+- Any other movable object that may travel the road network
+
+Pedestrians and animals are examples of traffic participants that are living beings.
+Vehicles are examples of traffic participants that are means of transportation.
+Therefore, the ego vehicle is also a traffic participant.
+
+The following figure shows the interface of a traffic participant.
+
+.. image:: _static/images/osi-traffic-participant-principle.png
+   :align: center
+   :alt: Traffic participant interfaces
+
+With every simulation step, an OSI traffic participant model receives ground truth data from the environment around itself, the sensor view.
+A traffic participant can output its own perceivable state, the traffic update.
+Traffic commands influence the behavior of the traffic participant model.
+Traffic commands allow event-based communication towards the traffic participant, for example, at certain simulation steps.
+OSI only specifies the interface of the traffic participant.
+OSI is agnostic to the architecture of the simulation environment and the traffic participant.
+
+
+Use cases for traffic participants
+----------------------------------
+
+Different models are involved in modeling a traffic participant.
+In all the use cases, a simulator loads and interprets a scenario and a map prior to execution.
+The scenario is, for example, provided by OpenSCENARIO.
+The map data is, for example, provided by OpenDRIVE.
+During runtime the simulator interacts with the traffic participants via OSI messages.
+There can be multiple instances of a traffic participant.
+The traffic participants are co-simulated.
+
+The following figure shows a very simple use case.
+
+.. image:: _static/images/osi-traffic-participant-use-case-1.png
+   :align: center
+   :alt: Simple traffic participant
+
+The traffic participant bases its behavior only on an idealized view of the area around it.
+If they exist, the traffic participant’s dynamics are included in the model.
+The map, the simulated world state, the sensor view, and the traffic updates represent the ground truth, meaning objective statements about the world.
+The scenario and the traffic commands on the other hand represent the intentions of the traffic participants.
```

Comment:
I would probably delete the last 2 sentences since you removed the colored clouds indicating the meaning of the messages. Without depicting this in the pictures I think this information is more confusing than helping understanding.

Reply:
- [ ] Delete last two sentences.